### ISSUE TITLE
Connection error when using HiSLIP

### ISSUE THREAD

**Issue by ckutlu** [Rohde-Schwarz/RsInstrument#25]
Hi there! Happy user of RsInstrument (and R&S oscilloscopes) here ;). I am trying to use hislip protocol when connecting to our R&S RTO6 oscilloscope.

What I am doing:

```python
rsc = RsInstrument("TCPIP::192.168.0.20::hislip0")
```
and I am getting:

> VisaIOError: VI_ERROR_NSUP_OPER (-1073807257): The given session or object reference does not support this operation.

If I pass explicitly "VxiCapable=False", the error disappears and I can use the instrument as expected. I guess being VXI capable and running on HiSLIP are mutually exclusive, so I'd expect not having to pass this option explicitly.

**Comment by Miloslav-RS** [comment]
Hi ckutlu, would you please share the IO Trace of the communication that produces your error?:
VisaIOError: VI_ERROR_NSUP_OPER (-1073807257): 

Something is not right, the HiSLIP is VXI-capable, this should work properly with our RTO6. I suspect another issue is in play here. Thanks for using the RsInstrument :-)

**Comment by okken** [comment]
Is this a duplication of #27 ?

**Comment by Miloslav-RS** [comment]
Yes, duplicate, the user did not respond. Closing the issue.

**Comment by ckutlu** [comment]
Hi @Miloslav-RS, apologies for not replying sooner. I went into vacation sometime in between and I ended up focusing on other stuff afterwards. I'd be happy to try out and share if you could you let me know how I could obtain IO Trace.

@okken I don't think this is a duplicate of #27 as the error symptoms are completely unrelated and I don't think we have enough evidence to conclude that the root cause is the same.

**Comment by Miloslav-RS** [comment]
Keeping this closed, since the solution is to install the newest pyvisa-py 0.8.0.

**Comment by ckutlu** [comment]
@Miloslav-RS While trying out #27, I gave this another try. I'm still getting an error, but the message is slightly different:

> RsInstrException: Operation Read STB: The given session or object reference does not support this operation.

If I pass, `VxiCapable=False` as an option, then the error disappears as before.

**Comment by Miloslav-RS** [comment]
I tested it again now, and it works even without the 'VxiCapable=False'.
What versions of PyVISA-py and PyVISA packages are you using?

**Comment by ckutlu** [comment]
@Miloslav-RS you are absolutely right! My `pyvisa-py` was 0.7.2 which was the latest supported by python 3.9. I updated my environment (for this test) to 3.11 and installed the latest pyvisa-py (0.8.0 released on April 1), the issue was resolved. Turns out `pyvisa-py` dropped support for 3.9 that's why my environment didn't pick up the latest one. We will also drop support for it soon, so this issue will be gone forever 🎉 .

Many thanks for your help!

**Comment by Miloslav-RS** [comment]
Perfect :-)
